docs/source/azure/examples/removepublicstorageips.rst

```diff
@@ -0,0 +1,22 @@
+Remove public storage IP-s
+==========================
+
+.. code-block:: yaml
+
+    policies:
+        - name: storage-no-public-ip
+        description: |
+            find publicly available storage ip-s and remove them keeping virtual network access intact
+        resource: azure.storage
+        
+        filters:
+        - type: value
+          key: properties.networkAcls.ipRules
+          value_type: size
+          op: ne
+          value: 0
+
+        actions:
+        - type: set-network-rules
+          default-action: Deny
+          ip-rules: []
```

Comment:
Does it keep 'bypass' property without any changes?

Reply:
It will set it to the default. I think it is possible to specify the current value, I just don't know the syntax of cloud custodian deep enough. Something like properties.networkAcls.bypass